src/lib/receiver.js

```diff
@@ -263,14 +290,18 @@ function createReceiver (opts) {
       return rejectIncomingTransfer(transfer.id, 'expired')
     }
 
+    if (protocol === 'psk' && !reviewPayment) {
+      debug('got PSK payment on non-PSK receiver')
+      return Promise.reject(new Error('opts.reviewPayment was not specified, so PSK is not supported on this receiver'))
```

Comment:
I think this should call `return rejectIncomingTransfer(transfer.id, 'psk-not-supported')` instead